Add CTA to look at roles library. (#881)
osohq/oso: docs/content/any/getting-started/_index.md

# Jump in

Learn the basics of using Oso and get Oso set up in your application.

> {{%ifLang "python" %}}

Author: ```suggestion
{{% ifLang "python" %}}
```

{{< callout "Early access to the next version of roles" "blue" >}}

We just released early access to the next version of built-in support

> for Roles. Initially supporting SQLAlchemy, this library, includes

Reviewer: ```suggestion
for Roles. Initially supporting SQLAlchemy, this library includes
```

Also `Initially supporting SQLAlchemy, this library...` feels like an unnecessarily oblique way to indicate that support for the new roles feature is only in the `sqlalchemy-oso` library.

Author: Yeah, I actually agree. Originally these snippets said 

> "Using SQLAlchemy?"

> We just released early access to the next version of our SQLAlchemy library, includes improved support for common authorization models, like role based access control.

Which IMO is clearer. Maybe I should just update the title but keep the contents the same?

The goal behind these changes (from @gneray) was to make roles the headline instead of SQLAlchemy for the early access program so that people interested in other languages maybe look at it and ask for support.

> {{< callout "Early access to the next version of roles" "blue" >}}
> 
> We just released early access to the next version of built-in support
> for Roles. Initially supporting SQLAlchemy, this library, includes
> improved support for common authorization models, like role based access
> control.
> 
> [Check it out here!](/new-roles)

Reviewer: @dhatch @gneray how about something like this

```suggestion
{{< callout "Built-in roles early access" "blue" >}}

We're hard at work on a major revision to Oso's built-in roles support.
We released [an early access version of our SQLAlchemy library](/new-roles)
containing the new features, which make it easy to model role-based access
control (RBAC) in Polar and enforce the policy over SQLAlchemy models.

[Check it out here!](/new-roles)
```

Author: We just discussed live a plan to:

1. Leave the SQLAlchemy callouts when the user has Python selected similar to as is (or maybe your suggestion)
2. Add callouts on roles pages for when users are not using Python that link to a higher level page describing features of the new roles work, with a link to try out in other languages that will email us.
